### DISCUSSION TITLE
College doesn't provide a Student Mail ID

### DISCUSSION THREAD

**Discussion by Arnav-Agarwal-01** [community/community#143555] [GitHub Education]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Greetings , I am a student of Keshav Memorial Institute Of Technology (KMIT) , India , and upon clarification with my college it has come to my knowledge that we are not going to be provided with student email id's and hence i am unable to access any of github's student developer packs or anything of that sort.
when i try to access it , it asks me for the name of my school and the name pops up , but then it asks me to add a mail account with the address kmit.edu.in , but i dont have any , and hence i am not able to move forward at any step.
Any help is appreciated.
Thankyou

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Arnav-Agarwal-01 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by Annapoornaaradhya** [comment]
Hello @Arnav-Agarwal-01 

Use a Personal Email: If your college allows it, try registering with a personal email (like Gmail or Yahoo) when accessing the Student Developer Pack. Some students have reported success using a personal email to verify their student status.

GitHub Education Support: You can contact GitHub Education support directly. Explain your situation, and they may have a solution for students without college-provided email addresses. You can reach them at [GitHub Support](https://support.github.com/contact).

Hope this helps you
